### PR TITLE
Move projects to employee namespace

### PR DESCRIPTION
**Pull request by Tivgres**

### CODE REVIEW

**Comment by Tivgres** [comment]
Don't need to merge it now.

File: app/controllers/employees_controller.rb [hunk 1]
```diff
@@ -9,6 +9,8 @@ def index; end
   def show
     @employee.build_image unless @employee.image
     @employee.projects.new.build_image
+    @project = Project.new
```
Comment (drkmen) [inline app/controllers/employees_controller.rb]: There is same line 11. We should create the project under employee so use @employee instance to build new project.

File: app/controllers/employees_controller.rb [hunk 2]
```diff
@@ -9,6 +9,8 @@ def index; end
   def show
     @employee.build_image unless @employee.image
     @employee.projects.new.build_image
```
Comment (drkmen) [inline app/controllers/employees_controller.rb]: @Tivgres here is new project has builded

File: app/controllers/employees_controller.rb [hunk 3]
```diff
@@ -9,6 +9,8 @@ def index; end
...
+    @project = @employee.projects.new
```
Comment (drkmen) [inline app/controllers/employees_controller.rb]: here to too
Why we build 2 projects?

**Pull request merged**